package.json

```diff
@@ -112,6 +112,6 @@
     "storybook": "start-storybook -p 6006",
     "build-storybook": "build-storybook",
     "cypress": "cypress open-ct",
-    "test:cypress": "cypress run-ct"
+    "cypress:ci": "cypress run-ct"
```

Comment:
Thanks. I'm a big fan of naming consistency (I heard it called "big endian" naming ... :) )